deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/Utils.java

```diff
@@ -24,4 +27,21 @@ static boolean hasEmptyOrNullString(String[] strings) {
     }
     return false;
   }
+
+  /** Strips off {@link #PARAM} and {@link #RETURN}, and removes line separator. */
+  static String formatJavaDoc(String str) {
+    if (str != null) {
+      str = str.trim().replace(System.lineSeparator(), " ");
```

Comment:
Removing line separators might make things harder to read, since we can't put the line breaks back. How about formatting as follows:

<deep link>
|#|
<Class/method>
|#|
<Javadoc>
|##|

So delimiter between each property would effectively be "\n|#|\n", and a delimiter between each element would be "\n|##|\n"